.github/workflows/playwright.yml

```diff
@@ -1,19 +1,196 @@
 name: Playwright Tests
 on:
-  workflow_run:
-    workflows: ["ci"]
-    types:
-      - completed
```

Comment:
Note to other reviewers that one of the main reasons this was written this way before was so PRs coming from forks could use the OAuth token secrets for e2e testing. We're going to try to avoid the need to use real tokens going forward, by having a test harness around external services, that the e2e test can act as and make assertions.